api/app/tests/fireweather_bot/test_hourly_actuals.py

```diff
@@ -1,33 +1,55 @@
 """ Unit testing for hourly actuals bot (Marvin) """
+from app.time_utils import get_utc_now
+import asyncio
 import os
 import logging
 import pytest
 from pytest_mock import MockerFixture
 from app.fireweather_bot import hourly_actuals
+from app.schemas.observations import WeatherReading, WeatherStationHourlyReadings
+from app.schemas.stations import WeatherStation
+import nest_asyncio
+nest_asyncio.apply()
```

Comment:
It patches the event loop to be nested, which I need to run async tests since event loop is already used in the call stack where this is called: https://pypi.org/project/nest-asyncio/